content/en/blog/2024/otel-docs-survey/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Insights from the OpenTelemetry Docs Usability Survey
+linkTitle: OTel-docs Survey
+date: 2024-12-13
+author: '[Tiffany Hrabusa](https://github.com/tiffany76) (Grafana Labs)'
+issue: https://github.com/open-telemetry/opentelemetry.io/issues/5793
+sig: Communications, End-User
+cSpell:ignore: Hrabusa
+---
+
+[The OpenTelemetry End-User SIG](/community/end-user/) recently surveyed the
+community to find out how user-friendly [OpenTelemetry's documentation](/docs/)
+is. In an earlier survey, two-thirds of respondents named comprehensive
+documentation as a top resource they wished they'd had when getting started with
+OpenTelemetry. So we decided to dig a little deeper. The Docs Usability Survey
+asked users where they go for OTel documentation, what they'd like to see more
+of in the docs, and how they rate the current state of the docs. We received 48
+responses, which we'll use to focus our documentation efforts and help us
+improve in key areas.
+
+Check out the results!
+
+## Key takeaways
+
+- Folks want more visual aids like diagrams and screenshots.
+- Of the three types of documentation we asked about (component concepts,
+  installation instructions, and troubleshooting), the troubleshooting docs need
+  the most work.
+- When asked what information they'd most like to see added to OTel's docs, the
+  top answers were **more examples** and **more coverage**, both in depth and
+  breadth.
+- The Collector docs were the most consulted docs set based on number of
+  ratings. This result tallies with the page view analysis in the SIG
+  Communications'
+  [year-end review](../year-in-review/#which-pages-were-the-most-popular).
+- After normalization and weighting, the
+  [Java documentation](/docs/languages/java/) was the overall highest rated docs
+  set, suggesting that the efforts made to
+  [improve its information architecture](../year-in-review/#ia-improvements)
+  have paid off. Meanwhile, the [Swift](/docs/languages/swift/) docs received
+  the overall lowest rating.
+- Among the six most popular docs sets, the [JavaScript](/docs/languages/js/)
+  docs received the lowest rating.
+
+## Detailed insights
+
+### About the respondents
+
+- 79% are using OTel in production.
+- 21% work for an observability or APM vendor.
+- 98% have previous knowledge of observability: intermediate (60%) or expert
+  (38%).
+
+#### Q: What source do you primarily rely on when you're looking for information about OpenTelemetry?
+
+- Overall, the majority of respondents (52%) rely on the opentelemetry.io
+  documentation.
+- Respondents early in their observability practice (beginner and intermediate)
+  are more likely to use the opentelemetry.io documentation.
+- Expert observability practitioners prefer the code repository documentation.
+
+![Percentage of website users by observability knowledge level](website-usage.png)
```

Comment:
How about replacing this image with markdown link this (it makes for a better viewing experience in dark mode):

```suggestion
> Respondents who use [opentelemetry.io] as their
> primary information source<br>_By level of observability knowledge_
>
> | **Beginners** | **Intermediates** | **Experts** |
> | :-----------: | :---------------: | :---------: |
> |     100%      |        62%        |     44%     |
```

Here's what it looks like:

> <img width="800" alt="image" src="https://github.com/user-attachments/assets/bfe1fbfb-7abc-4a9c-8cb8-0b4336d12775" />

Reply:
If you agree, then pls drop the corresponding image.

Reply:
Thanks. I didn't think about dark mode. I'll make the change.